lightning/src/ln/peer_handler.rs

```diff
@@ -459,6 +459,16 @@ impl<Descriptor: SocketDescriptor, CM: Deref, L: Deref> PeerManager<Descriptor,
 		}
 	}
 
+	/// Append a message to a peer's pending outbound/write buffer, and update the map of peers needing sends accordingly
+	fn enqueue_message<M: Encode + Writeable>(&self, peers_needing_send: &mut HashSet<Descriptor>, peer: &mut Peer, descriptor: Descriptor, message: &M) {
+		let mut encoded_message: Vec<u8> = vec![];
+		wire::write(message, &mut encoded_message);
```

Comment:
Was it intentional to not use `VecWriter` here? It tries to optimize by reserving space, but I noticed the implementation of `Write` for `Vec<u8>` already does that under the hood. Maybe we don't need `VecWriter` then.

Reply:
yes, though I can change it back to VecWriter for consistency's sake

Reply:
Honestly, I'd prefer to remove the other occurrence of `VecWriter` if it really isn't needed. Then we get both consistency and reduced complexity.